components/frontend-react/src/containers/AdminComponent.js

```diff
@@ -1,29 +1,29 @@
 import React from 'react';
 import Appbar from '../components/Appbar';
+import { connect } from 'react-redux';
+import { startAllDownloads, killAllDownloads } from '../actions/downloadActions'; 
 import StartKillDownloads from '../components/StartKillDownloads';
 import PendingSignupRequests from '../components/PendingSignupRequests';
 import UsageStats from '../components/UsageStats';
 
-import { requestsList } from '../stories/PendingSignupRequests.stories';
-import { usageStatsData } from '../stories/UsageStats.stories';
-
 const AdminComponent = (props) => {
-  const startDownloads = () => {
-    console.log('Start downloads');
-  };
-  
-  const killDownloads = () => {
-    console.log('Kill downloads');
-  };
-  
   return (
     <div>
       <Appbar data-test="component-appbar" isloggedIn={true} />
-      <StartKillDownloads data-test="component-start-kill-downloads" onStart={startDownloads} onKill={killDownloads} />
-      <PendingSignupRequests data-test="component-pending-signup-requests" requestsList={requestsList}/>
-      <UsageStats data-test="component-usage-statistics" data={usageStatsData}/>
+      <StartKillDownloads data-test="component-start-kill-downloads" onStart={props.startDownloads} onKill={props.killDownloads} />
```

Comment:
Let's destruct the props at first? 

```
  const { startDownloads, killDownloads } = props;
  return (
  ...
````

Reply:
@iamchathu @JaDogg @vivonk WDYT?

Reply:
@agentmilindu Agreed. Shall we add some `eslint` rules to enforce this ?

Reply:
Agree, Let's have `eslint` rule for this. @singhrohan62